base_layer/wallet/src/connectivity_service/service.rs

```diff
@@ -50,6 +50,12 @@ pub enum OnlineStatus {
     Offline,
 }
 
+impl Default for OnlineStatus {
```

Comment:
Think this `Default` impl should be removed

Reply:
There is a Default::default() when creating a new state, so this cannot be removed since #derive(default) is marked as experimental on enums. I have been explicit with OnlineStatus::Connecting though.